r/R/differential_expression.R

```diff
@@ -55,13 +54,19 @@ runDE <- function(req, data){
     message("checking meta.data slot ", str(data@meta.data))
 
     # Compute differential expression
-    result <- FindMarkers(data, group.by = "custom", ident.1 = "base", ident.2 = "background")
+    result <- presto::wilcoxauc(data, assay = "data", seurat_assay = "RNA",group_by="custom")
+    result <- result[result$group=="base",]
+    rownames(result) <- result$feature
+    result <- result[,c("pval","logFC","pct_in","pct_out","padj","auc")]
+    result$logFC <- log2(10^result$logFC)
+    colnames(result)<-list("p_val","avg_log2FC","pct_1","pct_2","p_val_adj","auc")
```

Comment:
Renaming the columns to maintain compatibility with the UI.

Reply:
`presto::wilcoxauc` returns `pct_in` and `pct_out` as actual percentages whereas `Seurat::FindMarkers` has columns `pct.1` and `pct.2` as fractions.